app/controllers/pages_controller.rb

```diff
@@ -2,9 +2,13 @@
 
 class PagesController < ApplicationController
   def view_issues
-    # If our location.js has pulled a location, use it to find Stops
-    if (params[:lat]) && (params[:long])
-      @nearby_stops = Stop.includes(:lines).within(0.2, origin: [params[:lat], params[:long]])
+	# If rails is not prod, just take the top 50 stops.
```

Comment:
It looks like you are using tabs for indents while rails (and the rest of this file) uses two space indents. Can you fix that? I'm surprised Rubocop didn't fail on that, so I might make some tweaks so CI will tell you that automatically

Reply:
Fixed this and set in my workspace settings in the meantime